pyproject.toml

```diff
@@ -1,3 +1,58 @@
+[build-system]
+requires = ["hatchling"]
+build-backend = "hatchling.build"
+
+[project]
+name = "django-stubs"
+version = "5.2.0"
+description = "Mypy stubs for Django"
+readme = "README.md"
+license = "MIT"
+license-files = ["LICENSE.md"]
+requires-python = ">=3.10"
+authors = [
+    { name = "Maksim Kurnikov", email = "maxim.kurnikov@gmail.com" },
+]
+maintainers = [
+    { name = "Marti Raudsepp", email = "marti@juffo.org" },
```

Comment:
```suggestion
    { name = "Marti Raudsepp", email = "marti@juffo.org" },
    { name = "Nikita Sobolev", email = "mail@sobolevn.me" },
```

Reply:
We should add other active maintainers as well, but I don't know their emails :(

- https://github.com/flaeppe
- https://github.com/adamchainz

Reply:
I've taken their emails from their public repos